src/output.zig

```diff
@@ -186,7 +186,7 @@ pub var stderr_descriptor_type = OutputStreamDescriptor.unknown;
 pub var stdout_descriptor_type = OutputStreamDescriptor.unknown;
 
 pub inline fn isEmojiEnabled() bool {
-    return enable_ansi_colors and !Environment.isWindows;
+    return enable_ansi_colors;
```

Comment:
oh lol. thats cool, now emojis work. we might have to do something later if conhost doesnt support emojis (or version checks to when conhost didn't)